queries.sql

```diff
@@ -30,3 +30,96 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 --Find all animals with a weight between 10.4kg and 17.3kg (inclusive):
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Begin the transaction
+BEGIN;
+
+-- Update the species column to "unspecified"
+UPDATE animals SET species = 'unspecified';
+
+-- Verify the change
+SELECT * FROM animals;
+
+-- Rollback the transaction
+ROLLBACK;
+
+-- Verify the species column reverted to its previous state
+SELECT * FROM animals;
+
+-- Begin the transaction
+BEGIN;
+
+-- Update the species column to "digimon" for animals with name ending in "mon"
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- Commit the transaction
+COMMIT;
+
+-- Verify the changes
+SELECT * FROM animals;
+
+-- Begin the transaction
+BEGIN;
+
+-- Update the species column to "pokemon" for animals without a species
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
+
+-- Commit the transaction
+COMMIT;
+
+-- Verify the changes
+SELECT * FROM animals;
```

Comment:
- [ ] This code here should be inside one single transaction as per requirements.

Screenshot of requirements
![image](https://github.com/tomdan-ai/Vet-clinic-Database/assets/81932478/0d46167a-c638-430d-bf07-d77268a9fa5d)
e.g
Your codebase should look something like this
```  
 BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```

### Note that you should update your screenshots to reflect this change